WolvenKit/Views/Shell/DockingAdapter.xaml.cs

```diff
@@ -274,8 +278,12 @@ private bool LoadLayout(string filePath, string layoutSource = "")
             {
                 _logger.Error(e);
             }
+            finally
+            {
+                reader?.Close();
```

Comment:
XmlReader is IDisposable, should probably Dispose it, or wrap in a using clause... Not sure why we're not doing it here

Reply:
not sure, I just moved the close into the finally case :D

Reply:
after work :) Feel free to take over!